go.mod

```
@@ -37,7 +37,7 @@ require (
 	github.com/tendermint/btcd v0.1.1
 	github.com/tendermint/crypto v0.0.0-20191022145703-50d29ede1e15
 	github.com/tendermint/go-amino v0.15.1
-	github.com/tendermint/iavl v0.13.4-0.20200621145059-83c3470ad61d
+	github.com/tendermint/iavl v0.13.4-0.20200622140716-a41a83e2415f
```

```suggestion
	github.com/cosmos/iavl v0.13.4-0.20200622140716-a41a83e2415f
```

Then `go mod verify` and `go mod tidy`

---

the go.mod in iavl has not migrated to `cosmos/iavl`. We should do this in the next major release of IAVL

---

```
go: github.com/cosmos/iavl@v0.13.4-0.20200622140716-a41a83e2415f: parsing go.mod:
	module declares its path as: github.com/tendermint/iavl
	        but was required as: github.com/cosmos/iavl
```